amethyst_assets/src/prefab/system.rs

```diff
@@ -89,8 +89,12 @@ where
             &**pool,
             strategy,
         );
-        prefab_handles
-            .populate_inserted(self.insert_reader.as_mut().unwrap(), &mut self.to_process);
+        prefab_handles.populate_inserted(
+            self.insert_reader
+                .as_mut()
+                .expect("`PrefabLoaderSystem::setup` was called before `PrefabLoaderSystem::run`"),
```

Comment:
I think you swapped the two methods here

Reply:
More specifically missing a `not` if I would write them similar as `amethyst_core` and earlier similar changes.